backend/de.metas.externalsystem/src/main/java/de/metas/externalsystem/rabbitmqhttp/interceptor/C_BPartner.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * #%L
+ * de.metas.externalsystem
+ * %%
+ * Copyright (C) 2021 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.externalsystem.rabbitmqhttp.interceptor;
+
+import de.metas.bpartner.BPartnerId;
+import de.metas.externalsystem.rabbitmqhttp.RabbitMQExternalSystemService;
+import de.metas.util.Services;
+import lombok.NonNull;
+import org.adempiere.ad.modelvalidator.annotations.Interceptor;
+import org.adempiere.ad.modelvalidator.annotations.ModelChange;
+import org.adempiere.ad.trx.api.ITrxManager;
+import org.compiere.model.I_C_BPartner;
+import org.compiere.model.ModelValidator;
+import org.springframework.stereotype.Component;
+
+@Interceptor(I_C_BPartner.class)
+@Component
+public class C_BPartner
```

Comment:
btw, is there a way to disable the whole rabbitMQ synchronization?

when and how this happens?
In case it happens elsewhere, isn't this an overhead here?

Reply:
cc @metas-ts

Reply:
good point..if there are no active external-system-configs (which is the case with most users!), the whole thing could be skipped

Reply:
added a check to RabbitMQExternalSystemService